loadgen/throughputstress/throughput_stress.go

```diff
@@ -12,14 +12,11 @@ type WorkflowParams struct {
 	Iterations int `json:"iterations"`
 	// If true, skip sleeps. This makes workflow end to end latency more informative.
 	SkipSleep bool `json:"skipSleep"`
-	// What iteration to start on. If we have continued-as-new, we might be starting at a nonzero
-	// number.
+	// What iteration to start on. If we have continued-as-new, we might be starting at a nonzero number.
 	InitialIteration int `json:"initialIteration"`
-	// If nonzero, we will continue as new after history has grown to be at least this many events.
-	ContinueAsNewAfterEventCount int `json:"continueAsNewAfterEventCount"`
+	// If nonzero, we will continue-as-new after specified iteration.
+	ContinueAsNewAfterIterCount int `json:"continueAsNewAfterIterCount"`
```

Comment:
I changed the way the workflow generates traffic here; this way resuming becomes much, much easier since the number of expected workflows can be calculated.